tests/integration.rs

```diff
@@ -5396,6 +5410,125 @@ mod tests {
                     .spawn()
                     .unwrap()
             }
+
+            // XXX Follow up test involving multiple disks will require:
+            // - Make image size variable
+            // - Make image filename random
+            // - Cleanup image file after test
+            // - Use NTFS instead of FAT, needs mkfs.ntsf in the container
+            fn disk_new(&self) -> String {
+                let img = PathBuf::from(
+                    String::from_utf8_lossy(b"/tmp/test-ntsf-hotplug-0.raw").to_string(),
```

Comment:
You typoed ntfs.

Reply:
Fixed the typo and added a better note about testing with NTFS, too.

thanks.